package.json

```diff
@@ -20,18 +21,18 @@
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "npm run test:lint && npm run test:unit && npm run data-validate",
-    "test:accessibility": "(lsof -t -i:7080 | xargs kill) && npm run build && run-p --race start bbc-a11y",
+    "test:accessibility": "npm run kill && npm run build && run-p --race start bbc-a11y",
     "test:accessibility:ci": "npm run build && run-p --race start bbc-a11y",
-    "test:e2e": "(lsof -t -i:7080 | xargs kill) && npm run build && run-p --race start cypress",
+    "test:e2e": "npm run kill && npm run build && run-p --race start cypress",
     "test:e2e:ci": "npm run build && run-p --race start cypress",
-    "test:e2e:interactive": "(lsof -t -i:7080 | xargs kill) && npm run build && run-p --race start cypress:interactive",
-    "test:e2e:storybook": "(lsof -t -i:9001 | xargs kill) && run-p --race storybook cypress:storybook",
```

Comment:
Some of these refer to the `9001` storybook port, whilst others refer to the 7080.